src/util/KeyHelper.ts

```diff
@@ -0,0 +1,76 @@
+/* tslint:disable:no-relative-imports */
+import { EncodeBuffer } from 'base-x'
+import * as bs58 from 'bs58'
+import { pki, random } from 'node-forge'
+
+import { SigningAlgorithm } from '../Interfaces'
+
+interface KeyPair {
+  readonly publicKey: string
+  readonly privateKey: string
+}
+
+interface PrivateKey {
+  readonly n: number
+  readonly e: number
+}
+
+const generateRsaPublicKeyFromPrivateKey = ({ privateKey }: { privateKey: PrivateKey }) =>
```

Comment:
Why wrap the privateKey in an object?

Reply:
This is a wrapper function to match signatures with ED25519's generatePublicKeyFromPrivateKey

Reply:
I mean in the argument, why not do 

```ts
const generateRsaPublicKeyFromPrivateKey = (privateKey: PrivateKey) =>
  pki.rsa.setPublicKey(privateKey.n, privateKey.e)

...

const getPublicKeyFromPrivateKey = (algorithm: SigningAlgorithm = SigningAlgorithm.Ed25519Signature2018) => (
  privateKey: number[]
): EncodeBuffer => {
  return Algorithms[algorithm].getPublicKeyFromPrivateKey(privateKey)
}
```

?

Reply:
`pki.ed25519.publicKeyFromPrivateKey` takes an options object `{ privateKey }` 

so 
```
return Algorithms[algorithm].getPublicKeyFromPrivateKey(privateKey) does not work
```
doesn't work unless I transform it.

I may be using